src/constants/validation.constants.ts

```diff
@@ -31,7 +31,7 @@ export const patterns = {
      * @example ValidationConstants.patterns.addressState.test("New York")
      * @example ValidationConstants.patterns.addressState.test("Québec")
      **/
-    addressState: /^[\p{L}\p{Nd}\s'.,(),-;]{0,100}$/u,
+    addressState: /^[\p{L}\p{Nd}\s'.,-;\[\]\(\)\$]{0,100}$/u,
```

Comment:
do we have $ in address ? 🤔

Reply:
As of i know we don't usually include $ in our address 🤔